AK/Coroutine.h

```diff
@@ -242,6 +242,12 @@ struct TryAwaiter {
         }
     }
 
+    void await_resume()
+    requires(IsSame<T, ErrorOr<void>>)
+    {
+        m_expression->release_value();
```

Comment:
`release_value` into nothingness? How about `.clear()`?

Reply:
Well `release_value` has/is a `has_value` assert, which I guess is preferable

Reply:
then please `(void)` it explicitly.

Reply:
Sure, done